src/toggle/toggle.stories.ts

```diff
@@ -66,12 +66,14 @@ storiesOf("Components|Toggle", module).addDecorator(
 	.addDecorator(withKnobs)
 	.add("Basic", () => ({
 		template: `
-			<a href="https://builder.carbondesignsystem.com/from-json/%7B%22title%22&#13;
-			%3A%22Toggle%22%2C%22data%22%3A%7B%22items%22%3A%5B%7B%22type%22&#13;
-			%3A%22toggle%22%2C%22header%22%3A%22Toggle%22%2C%22offText%22%3A%22Off%22%2C&#13;
-			%22onText%22%3A%22On%22%2C%22disabled%22%3Afalse%2C%22checked%22%3Afalse%2C&#13;
-			%22size%22%3A%22md%22%2C%22id%22%3A%222%22%2C%22codeContext%22%3A%7B%22&#13;
-			name%22%3A%22toggle-2%22%7D%7D%5D%2C%22id%22%3A1%7D%2C%22allCssClasses%22%3A%5B%5D%7D" target="_blank">
+			<a href="https://builder.carbondesignsystem.com/from-json/%7B%22id%22%3A&#13;
+			%2272575958133192826801788566988005%22%2C%22lastModified%22%3A%222023-03-10T16&#13;
+			%3A46%3A23.312Z%22%2C%22title%22%3A%22New%20fragment%22%2C%22data%22%3A%7B%22&#13;
+			items%22%3A%5B%7B%22type%22%3A%22toggle%22%2C%22header%22%3A%22Toggle%22%2C%22&#13;
+			offText%22%3A%22Off%22%2C%22onText%22%3A%22On%22%2C%22disabled%22%3Afalse%2C%22&#13;
+			checked%22%3Afalse%2C%22size%22%3A%22md%22%2C%22id%22%3A%222%22%2C%22codeContext%22&#13;
+			%3A%7B%22name%22%3A%22toggle-2%22%7D%7D%5D%2C%22id%22%3A1%7D%2C%22allCssClasses%22&#13;
+			%3A%5B%5D%7D" target="_blank">
```

Comment:
URL includes lastmodified & incorrect ID
```suggestion
			<a href="https://builder.carbondesignsystem.com/from-json/%7B%22id%22%3A%22ToggleModule%22%2C%22title%22%3A%22New%20fragment%22%2C%22data%22%3A%7B%22items%22%3A[%7B%22type%22%3A%22toggle%22%2C%22header%22%3A%22Toggle%22%2C%22offText%22%3A%22Off%22%2C%22onText%22%3A%22On%22%2C%22disabled%22%3Afalse%2C%22checked%22%3Afalse%2C%22size%22%3A%22md%22%2C%22id%22%3A%222%22%2C%22codeContext%22%3A%7B%22name%22%3A%22toggle-2%22%7D%7D]%2C%22id%22%3A1%7D%2C%22allCssClasses%22%3A[]%7D" target="_blank">
```